src/commands/cronjob/update.tsx

```diff
@@ -77,54 +86,39 @@ export default class Update extends ExecRenderBaseCommand<
       disable,
     } = this.flags;
 
-    const updateCronjobPayload: CronjobUpdateData = {};
+    timeout && checkTimeout(timeout.seconds);
 
     const destination = buildCronjobDestination(url, command, interpreter);
-    if (destination !== undefined) {
-      updateCronjobPayload.destination = destination;
-    }
-
-    if (enable) {
-      updateCronjobPayload.active = true;
-    } else if (disable) {
-      updateCronjobPayload.active = false;
-    }
-
-    if (description) {
-      updateCronjobPayload.description = description;
-    }
-
-    if (timeout) {
-      updateCronjobPayload.timeout = timeout;
-    }
-
-    if (email) {
-      updateCronjobPayload.email = email;
-    }
-
-    if (interval) {
-      updateCronjobPayload.interval = interval;
-    }
+    const updateCronjobPayload: CronjobUpdateData = {
+      ...(destination !== undefined && {
+        destination,
+      }),
+      ...(enable && { active: true }),
+      ...(disable && { active: false }),
+      ...(description && { description }),
+      ...(timeout && { timeout: timeout.seconds }),
+      ...(email && { email }),
+      ...(interval && { interval }),
+    };
```

Comment:
Personally, I find this harder to read (even if shorter) than the chain of `if`s before.

On a side note: If it's a simple matter of "if undefined, omit it from the payload", you can still simply (unconditionally) set `description: description`; if the value is set to `undefined`, it will be omitted from the generated JSON in any case (set it to `null`, if you explicitly want it to be in the JSON body).

This should be functionally equivalent:

```typescript
const updateCronjobPayload: CronjobUpdateData = {
  destination,
  active: enable ? true : (disable ? false : undefined), // note: maybe keep the if/else construct instead of the nested ternary for readability
  description,
  timeout: timeout?.seconds, // will evaluate to "undefined" when "timeout" is undefined
  email,
  interval,
};
```